llama-index-integrations/llms/llama-index-llms-fireworks/llama_index/llms/fireworks/base.py

```diff
@@ -0,0 +1,71 @@
+from typing import Any, Callable, Dict, Optional, Sequence
+
+from llama_index.core.base.llms.types import ChatMessage, LLMMetadata
+from llama_index.core.callbacks import CallbackManager
+from llama_index.core.constants import DEFAULT_NUM_OUTPUTS, DEFAULT_TEMPERATURE
+from llama_index.core.llms.generic_utils import get_from_param_or_env
```

Comment:
Should be `from llama_index.core.base.llms.generic_utils import get_from_param_or_env`